dev/devicelab/bin/run.dart

```diff
@@ -139,6 +140,8 @@ Future<void> _runTasks() async {
       await cocoon.sendTaskResult(builderName: luciBuilder, result: result, gitBranch: gitBranch);
     }
 
+    hostAgent.dumpFiles(result.detailFiles);
```

Comment:
Since this isn't dependent on flags, we should add this to `runTask` instead of `_runTasks`

Reply:
Thanks for the insight!